### ISSUE TITLE
conflicts with librocksdb-sys

### ISSUE THREAD

**Issue by idotial** [paritytech/rust-rocksdb#30]
when i add a dependency which depended on` rocksdb` to a project depended on ` kvdb-rocksdb`, it show error log while runing `cargo update`.
`error: failed to select a version for `parity-rocksdb-sys`.
    ... required by package `parity-rocksdb v0.5.0`
    ... which is depended on by `kvdb-rocksdb v0.1.4`
    ... which is depended on by `storage v0.0.1 (/Users/chenrui/Desktop/blockchain/WendingWorkStation/ipfs/filesys-rust/runtime/storage)`
    ... which is depended on by `beacon v0.0.1 (/Users/chenrui/Desktop/blockchain/WendingWorkStation/ipfs/filesys-rust/chain/beacon)`
    ... which is depended on by `client v0.1.0 (/Users/chenrui/Desktop/blockchain/WendingWorkStation/ipfs/filesys-rust/chain/client)`
    ... which is depended on by `alphanet v0.1.0 (/Users/chenrui/Desktop/blockchain/WendingWorkStation/ipfs/filesys-rust/chain/alphanet)`
    ... which is depended on by `devnet v0.1.0 (/Users/chenrui/Desktop/blockchain/WendingWorkStation/ipfs/filesys-rust/chain/devnet)`
    ... which is depended on by `filesys v0.1.0 (/Users/chenrui/Desktop/blockchain/WendingWorkStation/ipfs/filesys-rust)`
versions that meet the requirements `^0.5` are: 0.5.5, 0.5.4, 0.5.3, 0.5.2, 0.5.1, 0.5.0

the package `parity-rocksdb-sys` links to the native library `rocksdb`, but it conflicts with a previous package which links to `rocksdb` as well:
package `librocksdb-sys v5.17.2`
    ... which is depended on by `rocksdb v0.12.0`
    ... which is depended on by `filecoin-proofs v0.1.0 (https://github.com/idotial/rust-fil-proofs.git#53676171)`
    ... which is depended on by `filesys v0.1.0 (/Users/chenrui/Desktop/blockchain/WendingWorkStation/ipfs/filesys-rust)`

failed to select a version for `parity-rocksdb-sys` which could resolve this conflict`

**Comment by idotial** [comment]
https://github.com/idotial/testlinks.git here is a example project, run `cargo update` will show the error.

**Comment by dvdplm** [comment]
Hmm, `librocksdb-sys` is not the same as `parity-rocksdb-sys`. I don't think you can use both bindings at the same time. The formatting in your ticket is a little hard to follow, and the example is a little too minimal to understand what it is you're trying to achieve.
